include/types/MRSortModel.h

```diff
@@ -31,6 +31,7 @@ class MRSortModel {
    *
    * @param ap alternative performance object
    * @param id optional name of the model
+   * @param mode give the mode of the profile performance table
    */
```

Comment:
Looks like you should remove this changes as well. Btw can you update this doc to be the right one for this constructor? Looks like a copy paste mistake from my previous PR !